go-controller/pkg/controller-manager/controllerManager.go

```diff
@@ -0,0 +1,274 @@
+package controllerManager
```

Comment:
I would rename the files as well
`network-controller-manager/network_controller_manager.go`